inkstitch.py

```diff
@@ -2,27 +2,94 @@
 #
 # Copyright (c) 2010 Authors
 # Licensed under the GNU GPL version 3.0 or later.  See the file LICENSE for details.
-import cProfile
-import pstats
-import logging
+
 import os
 import sys
+import lib.debug_utils as debug_utils
+from pathlib import Path
+
+SCRIPTDIR = Path(__file__).parent.absolute()
+
+if len(sys.argv) < 2:
+    exit(1)  # no arguments - prevent accidentally running this script
+
+prefere_pip_inkex = True  # prefer pip installed inkex over inkscape bundled inkex
```

Comment:
This is just for backward compatibility with inkstitch. I understand that someone may need to test a new version of inkex library. But surely, it probably shouldn't be enabled by default. I also didn't understand why it's hardcoded there. It would certainly be better to have an option to enable it outside of the script, but where? Now there are already files like `DEBUG`, `PROFILE`. Should we add `PIP_INKEX`?

Or create a structured configuration file, for example, in **INI** format, e.g. `INKSTITCH_DEVEL.ini`, where all information for debug, profile, pip_inkex, and possibly other parameters will be stored.

Reply:
I think we really should always use the pip installed inkex rather than the inkscape delivered one for Ink/Stitch. The pip installed version is the version we will package with our releases and we want to run the very same version when we are developing Ink/Stitch. They often have updates that we want to use before the next Inkscape version is out.